### ISSUE TITLE
LocalStorageConsistencyChecking should accept 'Minimum' value

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#467]
When creating SPT and defining LocalStorage, the LocalStorageConsistencyChecking  parameter does not accept 'Minimum'value

### Steps to reproduce

```PowerShell
$name                               = "BFS-script"
$description                        = "VMWARE-STD-SY480GEN10-"
$spdescription                      = "esx-sss#"
$shtName                            = "SY 480 Gen10 1"
$sht                                = Get-HPOVServerHardwareType -Name $shtName
$egName                             = "EG_TUNNELED"
$eg                                 = Get-HPOVEnclosureGroup -Name $egName
$affinity                           = "Bay"
# -------------- Local Storage section
# -------------- Attributes for controller "Mezz 1" (Mixed)
$deviceSlot                         = "Mezz 1"
$controllerMode                     = "Mixed"
$controller1                        = New-HPOVServerProfileLogicalDiskController -ControllerID $deviceSlot -Mode $controllerMode -Initialize
# -------------- Attributes for controller "Embedded" (Mixed)
$deviceSlot                         = "Embedded"
$controllerMode                     = "Mixed"
$controller2                        = New-HPOVServerProfileLogicalDiskController -ControllerID $deviceSlot -Mode $controllerMode -Initialize
$controllers                        = $controller1, $controller2
$lsConsistency                      = "Minimum"
New-HPOVServerProfileTemplate -Name $name -Description $description -ServerProfileDescription $spdescription  -ServerHardwareType $sht -EnclosureGroup $eg -Affinity $affinity `
-LocalStorage  -StorageController $controllers -LocalStorageConsistencyChecking $lsConsistency
```
[2003200649_HPOV_transcript.log](https://github.com/HewlettPackard/POSH-HPOneView/files/4359938/2003200649_HPOV_transcript.log)

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.00.2368.2592](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.0.2368.2592) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2368.2592).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
